scripts/run_validator_node.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/bash
+
+sudo_cmd=""
+chain_spec=""
+keychain_exists=""
+session_keys=""
+
+echo "This script will setup a Atleta Validator on your PC. Press Ctrl-C at any time to cancel."
+echo "Checking privileges... "
+if [ $(id -u) -eq 0 ]; then
+	echo "OK (root)"
+elif command -v sudo &> /dev/null; then
+	echo "OK (sudo)"
+	sudo_cmd=sudo
+else
+	echo "FAIL"
+	echo "Not running as root and no sudo detected. Please run this as root or configure sudo. Exiting." >&2
+	exit 1
+fi
+
+echo "Detecting your architecture... "
+arch=$(uname -m)
+if [ "$arch" != "x86_64" -a "$arch" != "arm64" ]; then
+	echo "$arch is currently not supported by Atleta Node, only x86_64 or arm64 are supported. Exiting." >&2
+	exit 1
+else
+	echo "OK ($arch)"
+fi
+
+chain_spec_exists=0
+while [ "$chain_spec" == "" ]; do
+    echo "Available networks: "
+    i=0
+    echo "1) Devnet"
+    echo "2) Testnet"
+    echo "Select network for your running node: "
+    read choice < /dev/tty
+    case $choice in
+        1)
+            chain_spec="dev"
+            ;;
+        2)
+            chain_spec="testnet"
+            ;;
+        *)
+            echo "Invalid choice. Please select a right number."
+            continue
+            ;;
+    esac
+done
+
+echo "Your choose is: $chain_spec network."
+
+if [ "$(ls -A /opt/atleta/data/chains &>/dev/null)" ]; then
+	keychain_exists=1
+else
+	keychain_exists=0
+fi
+
+echo "Everything's ready. Tasks:"
+echo "  [X] Download atleta-node -> /usr/local/bin/atleta-node"
+echo "  [X] Download $chain_spec network chain spec -> /opt/atleta/chain_spec.json"
+if [ $keychain_exists -eq 0 ]; then
+	echo "  [X] Generate new session keys and store them in node"
+else
+	echo "  [ ] Data dir already exists, so session keys won't be regenerated."
+fi
+echo "Press Enter to continue or Ctrl-C to cancel."
+read < /dev/tty
+
+echo "Create directories..."
+$sudo_cmd mkdir -p /opt/atleta/ /usr/local/bin/
+echo "Stop old node if it's running..."
+
+#Check status of process
+if [[ $arch == "x86_64" ]]; then
```

Comment:
condition based only on architecture won't work as linux may be of both architectures. (not sure macos still can be intel, but it's better to check as well). you should rather check whether you are on linux or macos. also, with `systemctl` here we suppose that the user uses `systemd`, so we need to either make it a requirement or modify it so that it would work without `systemd`.